code/modules/recycling/disposal.dm

```diff
@@ -226,6 +227,11 @@
 		to_chat(user, SPAN_NOTICE("The opening is too narrow for [target] to fit!"))
 		return
 
+/// makes it so synths can't be flushed
+	if(U.isSynthetic() && (!isipc(U)))
```

Comment:
This seems overcomplicated.
Is there a reason to do that over a type check of U?